client/src/App.js

```diff
@@ -1,28 +1,55 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./style.css";
+import logo from './logo.svg';
+import './style.css';
+import Weather from './components/Weather'
 
 function App() {
-    return (
```

Comment:
As @Denijar has mentioned in the above comment, App.js should be changed in a separate PR. This PR should only focus on the Weather widget - you should revert changes outside of that widget to avoid affecting other PRs.  I noticed that you resolved the comment but this code is still being deleted at the moment.

Reply:
Ok, It should be the same with everyone's App.js now